tuxemon/core/db.py

```diff
@@ -86,11 +75,22 @@ def load(self, directory="all"):
 
         """
 
+        self.database = {
+            "item": {},
+            "monster": {},
+            "npc": {},
+            "technique": {},
+            "encounter": {},
+            "inventory": {},
+            "environment": {},
+        }
+
+        self.path = prepare.fetch("db")
```

Comment:
instance attributes shouldnt be delcared outside of init.  python lets you do that, but its not a good practice.  please move "databases" back to init, and you can set path to None there.  leaving instance variables in init helps developers learn about the class and what it does.

Reply:
Fair enough, I shouldn't have missed that. I put it back in init as None... the load() function must set it for the mod issue to be solved, so only its configuration is done there now.